app/src/lib/feature-flag.ts

```diff
@@ -26,9 +26,14 @@ function enableBetaFeatures(): boolean {
   return enableDevelopmentFeatures() || __RELEASE_CHANNEL__ === 'beta'
 }
 
+/** Should the new Compare view be enabled? */
+export function enableCompareBranch(): boolean {
+  return enableBetaFeatures()
+}
+
 /** Should PR integration be enabled? */
 export function enablePRIntegration(): boolean {
```

Comment:
I decided to leave this function around just in case we see something in the wild and want to back out.

Reply:
thoughts on adding a task to remove this later on at some point? one idea could be the 1.2 launch checklist? #3955?

Reply:
Eh, I'd do it before the 1.2 launch - :thumbsup: to `tech-debt` task.

Reply:
#4025